src/app/(routes)/mygallery/page.tsx

```
@@ -0,0 +1,92 @@
+  const [query, setQuery] = useState('');
+  const [selectedGrade, setSelectedGrade] = useState('');
+  const [selectedGenre, setSelectedGenre] = useState('');
+ 
+  return (
+    <div className="w-[1480px] mx-[240px]">
+      {/* Header */}
+      <div>
+        <Title
+          variant="primary"
+          font="baskin"
+          children="마이갤러리"
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)